### PR TITLE
A tutorial blog on how to deploy meshery on AKS

### LINKED ISSUE #2890: [Blog] Blog Post on Meshery on AKS
Contributors and community members are encouraged to post on https://layer5.io/blog. A new blog post fitting the following description is requested to be written by a community member. 

**Desired Title:** Meshery on AKS
**Topic:** This post is about how to setup and run Meshery on Azure K8s service (AKS)

**Core points to convey in the post:** 
1. Connect Meshery with AKS
1. Install Meshery on AKS Cluster
1. Access Meshery UI

---
<h4>Contributor Resources</h4>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by karnatisrinivas**
Signed-off-by: karnatisrinivas <karnatisrinivasulareddy@gmail.com>

**Description**

This PR fixes #2890 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 648ce1bc636ab39d433a9bb629dd9a75c3367c73 at: https://62c7539308517a0a7befbd6c--layer5.netlify.app

File: src/collections/blog/2022/2022-07-07-deploy-meshery-on-aks/index.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+az group create --name MesheryGroup --location southindia
+</code></pre>
+
+<p>Create AKS cluster using <code>az aks create</code>. The following command creates aks cluster with a single node. </p>
+
+<pre><code class="language-bash">
+az aks create --resource-group MesheryGroup --name MesheryAKS --node-count 1 --generate-ssh-keys
+</code></pre>
+<p>After a few minutes, the command completes and returns a JSON formatted information about the cluster.</p>
+<p>You can connect with your cluster by using <em>az aks get-credentials</em> ,  which basically downloads credentials and configure the Kubernetes CLI. </p>
```
Comment (debo19) [inline src/collections/blog/2022/2022-07-07-deploy-meshery-on-aks/index.mdx:37]:
```suggestion
<p>You can connect with your cluster by using <code>az aks get-credentials</code> ,  which basically downloads credentials and configure the Kubernetes CLI. </p>
```

File: src/collections/blog/2022/2022-07-07-deploy-meshery-on-aks/index.mdx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+<pre><code>
+az aks get-credentials --resource-group MesheryGroup --name MesheryAKS
+</code></pre>
+<p>Verify the connection to your cluster using the kubectl get command. </p>
```
Comment (debo19) [inline src/collections/blog/2022/2022-07-07-deploy-meshery-on-aks/index.mdx:41]:
```suggestion
<p>Verify the connection to your cluster using the <code>kubectl get</code> command. </p>
```

File: src/collections/blog/2022/2022-07-07-deploy-meshery-on-aks/index.mdx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+published: true
+resource: true
+---
+
+import { BlogWrapper } from "../../Blog.style.js";
+import mesheryui from "./mesheryui.png";
+
+<BlogWrapper>
+
+<p><em>Meshery is the open-source, cloud native management plane that can provision ten different service meshes, onboard your applications, manage WebAssembly filters, apply service mesh patterns, validate against best practices, and benchmarks the performance of your service mesh deployments.</em></p>
```
Comment (debo19) [inline src/collections/blog/2022/2022-07-07-deploy-meshery-on-aks/index.mdx:21]: Link to `/meshery`

**Comment by l5io** [comment]
🚀 Preview for commit a99bb64ddf3eac5a4edef4673ec1bd1fed9556c8 at: https://62cbd54b368e156805e22458--layer5.netlify.app

**Comment by leecalcote** [comment]
Cool! 😎

File: src/collections/blog/2022/2022-07-07-deploy-meshery-on-aks/index.mdx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+resource: true
+---
+
+import { BlogWrapper } from "../../Blog.style.js";
+import mesheryui from "./mesheryui.png"; 
+
+<BlogWrapper>
+
+<p><em><a href="https://meshery.io/">Meshery</a>  is the open-source, cloud native management plane that can provision ten different service meshes, onboard your applications, manage WebAssembly filters, apply service mesh patterns, validate against best practices, and benchmarks the performance of your service mesh deployments.</em></p>
+<p><em>Meshery’s vision is to make the operation of cloud native infrastructure and the service mesh layer of cloud simplified. Originally created by Layer5, Meshery is an open source project with hundreds of contributors world-wide and is actively maintained by engineers from Red Hat, VMware, Intel, Layer5 and others. </em></p>
```
Comment (leecalcote) [inline src/collections/blog/2022/2022-07-07-deploy-meshery-on-aks/index.mdx:22]:
```suggestion
<div class="intro">
<p><a href="https://meshery.io/">Meshery</a>'s goal is to make the operation of cloud native infrastructure and the service mesh layer of cloud simplified. Originally created by Layer5, Meshery is an open source project with hundreds of contributors world-wide and is actively maintained by engineers from Red Hat, VMware, Intel, Layer5 and others.</p>
</div>
```

**Review by leecalcote — APPROVED** [review]
Good stuff, @karnatisrinivas 👍

**Pull request merged**